### PR TITLE
Rajarju/aio where output

### PR DESCRIPTION
**Pull request by rajarju**
## Related Issue

Fix: #115 
Fix: #118 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rajarju** [comment]
[ ] Need to check if the Error should be thrown if org is not selected

File: test/commands/console/index.test.js [hunk 1]
```diff
@@ -71,8 +73,11 @@ describe('ConsoleCommand', () => {
           return null
         })
         command.printConsoleConfig()
-        expect(command.log).toHaveBeenCalledTimes(1)
-        expect(command.log).toHaveBeenCalledWith(noOrgErrorMessage)
+        expect(command.log).toHaveBeenCalledTimes(4)
+        expect(command.log).toHaveBeenCalledWith('You are currently in:')
+        expect(command.log).toHaveBeenCalledWith('1. Org: <no org selected>')
+        expect(command.log).toHaveBeenCalledWith('2. Project: THE_PROJECT')
```
Comment (shazron) [inline test/commands/console/index.test.js:79]: not sure this makes sense, how can you select a project without an org being selected? I get that this tests a permutation, but the config shouldn't ever get to this state (can the test be removed instead?)

File: test/commands/console/index.test.js [hunk 2]
```diff
@@ -82,8 +87,11 @@ describe('ConsoleCommand', () => {
           return null
         })
         command.printConsoleConfig()
-        expect(command.log).toHaveBeenCalledTimes(1)
-        expect(command.log).toHaveBeenCalledWith(noOrgErrorMessage)
+        expect(command.log).toHaveBeenCalledTimes(4)
+        expect(command.log).toHaveBeenCalledWith('You are currently in:')
+        expect(command.log).toHaveBeenCalledWith('1. Org: <no org selected>')
+        expect(command.log).toHaveBeenCalledWith('2. Project: <no project selected>')
+        expect(command.log).toHaveBeenCalledWith('3. Workspace: THE_WORKSPACE')
```
Comment (shazron) [inline test/commands/console/index.test.js:94]: same here. for a workspace to be selected, project and org needs to be selected. I get that this tests a permutation, but the config shouldn't ever get to this state (can the test be removed instead?)

File: test/commands/console/index.test.js (hunk 1 above)
Comment (rajarju) [reply test/commands/console/index.test.js:79]: I agree, I will fix it along with #118

**Comment by rajarju** [comment]
@shazron Updated PR

**Pull request merged**